src/main/java/com/gladguys/alucoapi/entities/StudentClassObservation.java

```diff
@@ -0,0 +1,27 @@
+package com.gladguys.alucoapi.entities;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Data;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Data
+@Entity(name = "student_class_observation")
+public class StudentClassObservation {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "student_id")
+    private Student student;
+
+    @ManyToOne
+    @JoinColumn(name = "class_id")
+    private Class classStudent;
+
+    @JsonFormat(pattern = "dd/MM/yyyy")
+    private LocalDate date;
+    private String observation;

```

Comment:
se a observação for grande, pode da problema pro tamanho do campo. usar @Column(columnDefinition = "text")